_component_design/popovers.md

```diff
@@ -5,90 +5,180 @@ secondary: Editing
 layout: component
 category: Components
 usage: >
-  A popover is a small group of information or inputs that is presented in a panel that is overlayed on top of the original page, preserving the user’s context.
+  To preserve the user’s context, popovers present a small group of information or inputs that is presented in a panel that is overlayed on top of the original page, preserving the user’s context.
 preview-image: preview-images/popover.svg
 resource: true
-status: unknown
-last-modified: 2017-08-17
+status: in-progress
+last-modified: 2017-10-22
 ---
 
 {% include toc.html %}
 
-Popovers are triggered as a result of a user action and provide a user with
-additional context through language, actions through buttons, or both. Popovers
-come in a the following types Hover Popover, Triggered Popover, and Closeable
-Popover
-
-# Hover Popover
+## Introduction
 
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-Hover popovers are triggered as a result of a user hovering over linked area in
-the UI.
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to Use
+{: .hxSubSectionTitle}
+Popovers are triggered as a result of a user action and provide a user with additional context through language, actions through buttons, or both. 
+
+Popovers may also be used for short create or edit workflows and are particularly useful if context from the page is needed while making the change. If a large workflow is required, consider creating a new page.
+
+
+### Best practices
+
+
+- Avoid opening a popover from within another popover
+- Open popovers via click, and never open them via hover.
+- Avoid putting important or required information in a popover because the content inside of a popover isn't always visible.
+- Use popovers to make small changes with a small number of inputs.
+
+### Problems being solved
+
+- For users needing to make changes or take actions while remaining in their original context (for example, a page), a popover allows for small changes or actions to take place without leaving the page.
+- For users needing to be able to view additional details about an item in a workflow, a popover allows easy access to this complex or interactive information without cluttering the page.
```

Comment:
Popovers give easy, uncluttered access to additional details about an item in a workflow.

(some of this is getting repetitive. Don't be afraid to cull.)

Reply:
Agreed. Combined the two bullet points into a single sentence, now reads: "- Popovers allow users to make small changes, view additional details, or take actions without leaving the page, which preserves context. In addition to preserving context, using popovers declutters a webpage."